polkadot-protocol/architecture/polkadot-chain/agile-coretime.md

```diff
@@ -1,4 +1,48 @@
 ---
 title: Agile Coretime
-description: TODO
----
\ No newline at end of file
+description: Explore the efficient scheduling mechanisms to access Polkadot cores to produce blockspace continuously or on-demand.
+---
+
+# Agile Coretime
+
+Agile Coretime is the [scheduling](https://en.wikipedia.org/wiki/Scheduling_(computing)){target=\_blank} 
+framework on Polkadot that lets parachains efficiently access cores, which comprise an 
+active validator set tasked with parablock validation. Polkadot is the first blockchain to
+enable such a flexible scheduling framework for producing blockspace. The cores can be designated
+to a parachain continuously through [bulk coretime](#bulk-coretime) or [on-demand](#on-demand-coretime).
+It is also possible to schedule multiple cores in parallel through elastic scaling, which is
+a feature under active development on Polkadot. 
+
+## Bulk Coretime
+
+Bulk coretime is a fixed duration of continuous coretime represented by an NFT that can be 
+purchased through [coretime sales](#coretime-sales) in DOT and can be split, shared, or 
+resold. Currently, the duration of bulk coretime is set to 28 days. Coretime purchased in bulk and assigned to a single parachain
+is eligible for a price-capped renewal, providing a form of rent controlled access which is 
+important for predicting the running costs in the near future. If the bulk coretime is 
+[interlaced](#coretime-interlacing) or [split](#coretime-splitting) or is kept idle without
+assigning it to parachain, it will be ineligible for for the price-capped renewal.
```

Comment:
```suggestion
Bulk coretime is a fixed duration of continuous coretime represented by an NFT that can be 
purchased through [coretime sales](#coretime-sales) in DOT and can be split, shared, or 
resold. Currently, the duration of bulk coretime is set to 28 days. Coretime purchased in bulk and assigned to a single parachain is eligible for a price-capped renewal, providing rent-controlled access, which is 
important for predicting the running costs shortly. If the bulk coretime is 
[interlaced](#coretime-interlacing) or [split](#coretime-splitting) or is kept idle without
assigning it to parachain, it will be ineligible for for the price-capped renewal.
```